pkg/cloud/services/elb/loadbalancer_test.go

```diff
@@ -228,6 +228,20 @@ func TestGetAPIServerClassicELBSpec_ControlPlaneLoadBalancer(t *testing.T) {
 				}
 			},
 		},
+		{
+			name: "load balancer config with elb health check protocol specified",
+			lb: &infrav1.AWSLoadBalancerSpec{
+				HealthCheckProtocol: &infrav1.ClassicELBProtocolTCP,
+			},
+			mocks: func(m *mock_ec2iface.MockEC2APIMockRecorder) {},
+			expect: func(t *testing.T, res *infrav1.ClassicELB) {
+				t.Helper()
+				expectedTarget := fmt.Sprintf("%v:%d", infrav1.ClassicELBProtocolTCP, 6443)
+				if res.HealthCheck.Target != expectedTarget {
+					t.Errorf("Expected health check target to be %v, got %v", expectedTarget, res.HealthCheck.Target)
+				}
+			},
+		},
 	}
```

Comment:
Can you write a use case where
```
lb: &infrav1.AWSLoadBalancerSpec{}
```
In this case you need to call webhooks default function

Reply:
Can I add this check in one of the existing test case? to reduce duplication?

Reply:
Done